Tools/read_raw_data.py

```diff
@@ -297,4 +297,4 @@ def _read_buffer(snapshot, header_fn, _component_names):
 
 
 if __name__ == "__main__":
-    data = read_lab_snapshot("lab_frame_data/snapshot00012");
+    data = read_lab_snapshot("lab_frame_data/snapshot00012", "lab_frame_data/Header");
```

Comment:
Thanks for this PR!
I think that "lab_frame_data/Header" is still a global (i.e. for all snapshots) header.
The one specific for "snapshot00012" would be in:
`lab_frame_data/snapshot00012/Header`.
As an example, I am attaching here the header file I find inside snapshot00012 for the 3D example using the boosted frame for PWFA:
[SN12_Header.txt](https://github.com/ECP-WarpX/WarpX/files/3942255/SN12_Header.txt)
And the global header file for all snapshots:
[Gl_Header.txt](https://github.com/ECP-WarpX/WarpX/files/3942264/Gl_Header.txt)

Is this PR related to any issue?

Reply:
This is just an example that is not working. I spotted via an alert in LGTM:
https://lgtm.com/projects/g/ECP-WarpX/WarpX/alerts/?mode=list&severity=error

Since this is just an example how to use this script, I trust your recommendation what to put here - global or snapshot-local header :)